packages/core/src/Stores/notification-store.js

```diff
@@ -369,6 +374,17 @@ export default class NotificationStore extends BaseStore {
                 this.addNotificationMessage(this.client_notifications.close_mx_mlt_account);
             }
 
+            if (
+                is_financial_assessment_incomplete &&
+                is_fully_authenticated &&
+                content_flag === ContentFlag.LOW_RISK_CR_NON_EU &&
```

Comment:
Shouldnt we show this notification for all CR clients?
LOW_RISK_CR_NON_EU is only for DIEL clients. can you double check this?